Example/Tests/Classes/LeanplumTest.m

```diff
@@ -1122,6 +1166,70 @@ - (void)test_variables
     XCTAssertEqualObjects(@100LL, [long_long_variable defaultValue]);
 }
 
+/**
+ * Tests variant debug info.
+ */
+- (void)testStartResponseShouldParseVariantDebugInfo
+{
+    [Leanplum setVariantDebugInfoEnabled:YES];
```

Comment:
test style: adding a few comments for `given/when/then` will make this test much easier to understand and maintain in the future